src/main/java/com/crudtest/controller/UserController.java

```diff
@@ -30,53 +33,97 @@ public void initBinder(WebDataBinder webDataBinder) {
         webDataBinder.addValidators(userValidator);
     }
 
-    @RequestMapping(value = "/", method = RequestMethod.GET)
+
+    @RequestMapping(value = "/register", method = RequestMethod.GET)
     public ModelAndView addUser() {
         ModelAndView modelAndView = new ModelAndView();
         UserForm form = new UserForm();
         modelAndView.addObject("form", form);
-        modelAndView.setViewName("index");
+        modelAndView.setViewName("register");
         return modelAndView;
     }
 
-    @RequestMapping(value = "/", method = RequestMethod.POST)
+    @RequestMapping(value = "/register", method = RequestMethod.POST)
     public ModelAndView addUser(@ModelAttribute("form") @Valid UserForm form, BindingResult bindingResult) {
         ModelAndView modelAndView = new ModelAndView();
 
-        if (userService.findByEmail(form.getEmail()) != null) {
```

Comment:
Since we are not using a facade layer I would recommend to move the handling of the binding result and the model and view to the user service so the controller is cleaner.